pyrios-ui/manifests/deployment.yaml

```diff
@@ -32,10 +32,10 @@ spec:
     spec:
       containers:
       - name: pyrios-ui
-        image: gcr.io/pso-examples/pyrios-ui:1.0.1
+        image: gcr.io/pso-examples/pyrios-ui:latest
```

Comment:
yah no latest please.  Not sure how to version easily, but we need to version

Reply:
I have a plan for versioning and will get it into next iteration!